src/libs/actions/Policy.ts

```diff
@@ -200,6 +203,18 @@ Onyx.connect({
     callback: (val) => (allRecentlyUsedTags = val),
 });
 
+const allCategoryPolicies: OnyxCollection<PolicyCategories> = {};
+Onyx.connect({
+    key: ONYXKEYS.COLLECTION.POLICY_CATEGORIES,
+    callback: (val, key) => {
+        if (!key) {
+            return;
+        }
+        const policyID = CollectionUtils.extractCollectionItemID(key);
+        allCategoryPolicies[policyID] = val;
+    },
```

Comment:
I think we can simplify this by:

```suggestion
    callback: val => allPolicyCategories = val
    waitForCollectionCallback: true,
```